### PR TITLE
travis: bump to llvm toolchain 6.0

### PR DESCRIPTION
**Pull request by bagder**

### CODE REVIEW

**Comment by MarcelRaad** [comment]
If your goal is to use clang 6, you also have to update the package version as currently clang 3.9 is still used. For this, you have to wait for (or incorporate) https://github.com/curl/curl/pull/2476 which I want to merge tomorrow morning. And while you're at it, you could also update GCC to 7 then :-)

**Comment by bagder** [comment]
Yes, that's exactly where I want to go... =)

**Comment by bagder** [comment]
I'll await your merge of that, I'll just check that everything else seems to be fine in the mean time...

File: .travis.yml [hunk 1]
```diff
@@ -145,7 +145,7 @@ script:
         # Uncomment this when `coverage` runs on Trusty.
         # set -eo pipefail
         if [ "$T" = "coverage" ]; then
-             export CC="gcc-6"
+             export CC="gcc-7"
```
Comment (MarcelRaad) [inline .travis.yml]: When I updated the GCC version in https://github.com/curl/curl/commit/828187d86b70af3a031b529312b8027b4ece10a5, I had to change the gcov version too.

Comment (bagder) [reply .travis.yml]: Ah yes, I *suspected* this as well... thanks!

**Pull request closed without merging**